examples/browse.rs

```diff
@@ -1,13 +1,33 @@
 use astro_dnssd::browser::*;
+use env_logger::Env;
+use log::{error, info};
+use std::net::ToSocketAddrs;
 
 fn main() {
+    env_logger::from_env(Env::default().default_filter_or("trace")).init();
+    info!("Starting browser...");
     let mut browser = ServiceBrowserBuilder::new("_http._tcp").build().unwrap();
     let _result = browser.start(|result| match result {
-        Ok(service) => println!(
-            "Reply: if: {} name: {} type: {} domain: {}",
-            service.interface_index, service.name, service.regtype, service.domain
-        ),
-        Err(e) => println!("Error: {:?}", e),
+        Ok(mut service) => {
+            let event = match service.event_type {
+                ServiceEventType::Added => "Added",
+                ServiceEventType::Removed => "Removed",
+            };
+            info!(
+                "{}: if: {} name: {} type: {} domain: {}",
+                event, service.interface_index, service.name, service.regtype, service.domain
+            );
+            let results = service.resolve();
+            for r in results.unwrap() {
+                let path = r.txt_record.as_ref().unwrap().get("s");
```

Comment:
Should we maybe add a quick comment here about the significance of "s"?  Not sure I know what it is at first glance.

Reply:
yeah maybe, it's just to test, probably better to spell out a key :)